seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/source/JdbcSource.java

```diff
@@ -163,10 +163,10 @@ private PartitionParameter initPartitionParameter(String columnName, Connection
             if (rs.next()) {
                 max = jdbcSourceOptions.getPartitionUpperBound().isPresent() ?
                     jdbcSourceOptions.getPartitionUpperBound().get() :
-                    Long.parseLong(rs.getString(1));
+                    rs.getString(1) == null ? 0L : Long.parseLong(rs.getString(1));
```

Comment:
I suggest use `Long.MAX_VALUE` and `Long.MIN_VALUE` when `rs.getString(1) == null` or `rs.getString(2) == null `. And then we should add warning log too.

Reply:
done.